### PR TITLE
[Docs] Make sure venv is py3 and export app

### PR DESCRIPTION
**Pull request by hughhhh**
- [X] Update docs to install a virtualenv for python3
- [X] Export app before starting flask app

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ superset init
 superset load_examples
 
 # Start the Flask dev web server (but see below for frontend asset compilation)
+export FLASK_APP=superset/__init__.py # you only have to do this once
```
[the remark this answers is not in the document]
Comment (hughhhh) [reply CONTRIBUTING.md]: So going through the install doc ran into this error for exporting the `FLASK_APP` var, should we explicitly say `pip install python-dotenv`?

[the remark this answers is not in the document]
Comment (mistercrunch) [reply CONTRIBUTING.md]: I added it to `requirements-dev.txt`, I think that one is mentioned on the CONTRIBUTING.md

[the remark this answers is not in the document]
Comment (hughhhh) [reply CONTRIBUTING.md]: ok removing this line then

**Comment by mistercrunch** [comment]
I wonder if all systems and linux distros have a `python3` link in their path. That may not be the case moving forward as it becomes the default. `python3` may also be unsupported versions of python (3.4, 3.5, ...). Maybe we should add a note or something there "Setup a python3.6 virtualenv"

**Pull request merged**